### DISCUSSION TITLE
@vercel/lambda-node-bridge error on next 13.1.6 version

### DISCUSSION THREAD

**Discussion by kumard3** [vercel/next.js#45753] [Help]
so I have just updated my next version to the latest and after deploying it to vercel I am having this error 
![Screenshot 2023-02-09 at 10 14 16 PM](https://user-images.githubusercontent.com/62591080/217883266-ec13828c-a9bd-4761-b16a-33051ca46743.png)

**Comment by ijjk** [comment]
Hi, this should now be resolved, please redeploy your project!
